styles/angular-patternfly.less

```diff
@@ -15,4 +15,4 @@
 @import "../src/canvas-view/canvas-editor/canvas-editor.less";
 @import "../src/pagination/pagination.less";
 @import "../src/datepicker/datepicker.less";
-
+@import "../src/modals/modal-overlay/modal-overlay.less";
```

Comment:
Don't we need the equivalent for SASS?

Reply:
SASS is auto-created.

Reply:
not committed?

Reply:
Based on angular-patternfly's README.md, I believe it is automatically handled as part of the PR: "The Less to Sass Conversion step will be accomplished and managed as a part of any Pull Request which includes Less file changes. Although contributors may want to build and test their style changes with Sass before submitting a Pull Request, this step should always be tested and validated by reviewers before a style change is merged and released. "
..still looking to verify what exactly a-pf's TravisCI build cmd is to see if it does a '--sass'

Reply:
OK, looks like it is picking it up automatically. Thought you'd need to add the new scss file to the imports but looks like we are smaht enough to pick it up.

Reply:
Hi, @jeff-phillips-18 you are correct, in the README.md "Note: When a Less file is added/deleted/renamed it needs to be updated in the main Less import file /styles/angular-patternfly.less and the main Sass import file styles/_angular-patternfly.scss."
@nicolethoen, I think you need to also include the new .less file in `styles/_angular-patternfly.scss`.

Reply:
Ah, but it is not added to the dist/sass/_angular-patternfly.scss file so it won't be automatically included for applications that simply import the single file rather than all the scss files.

Please add the import to https://github.com/patternfly/angular-patternfly/blob/master/styles/_angular-patternfly.scss